src/Apps/Checkout/checkoutRoutes.tsx

```diff
@@ -30,6 +38,18 @@ export const checkoutRoutes: RouteProps[] = [
         path: "",
         getComponent: () => CheckoutOverviewRoute,
       },
+      {
+        path: ":orderID",
+        Component: ApplePayRoute,
+        prepareVariables: params => ({ orderID: params.orderID }),
+        query: graphql`
+          query checkoutRoutes_ApplePayQuery($orderID: ID!) {
+            order: commerceOrder(id: $orderID) {
+              ...ApplePay_order
+            }
+          }
+        `,
```

Comment:
I'm not sure about the difference in routing terms between this and the previous one with no order id (do we need it the OverviewRoute any more?). Technically i would avoid the apple pay name since this is the express checkout element, and I would keep it all in one route component even if this specific element needs to be rendered conditionally. However I don't think it's blocking for now.

Reply:
I believe the OverviewRoute is kind of a place to test the payment element, while this strictly requires an order id, not sure if we want both later on, since this component is only a test component to test Apple Pay only

Reply:
I would suggest that we keep this one as is right now (maybe merge it?), and add a comment to remove it after the testing is done by Stripe? What do you think @erikdstock?